### ISSUE TITLE
improve how it finds the Marian binary

### ISSUE THREAD

**Issue by frankseide** [marian-nmt/marian-regression-tests#31]
I suggest to update the default location for the marian binary.
```
~/marian-dev> regression-tests/run_mrt.sh
Error: '/home/fseide/build/build/marian' not found. Do you need to compile the toolkit first?
```
We now have a submodule reference inside the Marian build directory, so we should assume that that is the main use, and therefore the default should in my view be something `directory of run_mrt.sh script/../build/marian`.

**Comment by snukky** [comment]
Fixed (9319cd39c37771f84d71523d539fb4f7a511269b). The change will take effect with the next update of submodule in marian-dev. Thanks.

**Comment by snukky** [comment]
The submodule has been updated.
